package-lock.json

```diff
@@ -7374,6 +7405,108 @@
 				"rememo": "^3.0.0",
 				"tinycolor2": "^1.4.1",
 				"uuid": "^3.3.2"
+			},
+			"dependencies": {
```

Comment:
Can you try to remove this section manually and run `npm i` again? :)

Reply:
@gziolo Sure thing!